packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/errors/call-stack/call-stack.tsx

```diff
@@ -50,19 +47,18 @@ export function CallStack({ frames, dialogResizerRef }: CallStackProps) {
 
     if (isIgnoreListOpen) {
       function onTransitionEnd() {
-        setIsIgnoreListOpen(false)
         dialog.removeEventListener('transitionend', onTransitionEnd)
+        setIsIgnoreListOpen(false)
       }
       dialog.style.height = `${initialDialogHeight.current}px`
       dialog.addEventListener('transitionend', onTransitionEnd)
     } else {
-      dialog.style.height = `${initialDialogHeight.current + ignoreListHeight}px`
-      setIsIgnoreListOpen(!isIgnoreListOpen)
```

Comment:
This just simplifies the logic, we don't need to add the height manually, parent `ResizeObserver` will do its job